ui/components/thing-counters.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+export default function ThingCounter({ count, singular, plural }, { router }) {
+  const query = router.location.query;
+  const queryKeys = Object.keys(query);
+  const noun = count === 1 ? singular : plural;
+  const verb = count === 1 ? 'matches' : 'match';
+  const classes = count === 0 ? 'alert p1 m1 border' : '';
```

Comment:
I think we'll need to include the associated SASS edits, too: https://github.com/18F/omb-eregs/pull/416/files#diff-a17de2846fcbead1087a20fc2730d26d